tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/types/TupleTypeBalaTest.java

```diff
@@ -85,6 +93,40 @@ public void testTupleTypeNegative() {
         Assert.assertEquals(negativeResult.getErrorCount(), i);
     }
 
+    @Test
+    public void testTupleAnnotations1() {
+        Object returns = BRunUtil.invoke(result, "testTupleAnnot1", new Object[]{});
+
+        Type t1 = TypeUtils.getType(returns);
+        Object annot = ((BAnnotatableType) t1).getAnnotation(StringUtils.fromString("$field$.1"));
+
+        BMap<BString, Object> expected = ValueCreator.createMapValue();
+        expected.put(StringUtils.fromString("tuples/tuple_type_project:0:member"), true);
```

Comment:
More accurate bala test would be to assert the values at compile-time, like in org.ballerinalang.test.bala.annotation.AnnotationTests#testParamAnnotAttachmentsViaBir.

Reply:
Should the annotation be attached to the type here or only exist in the annotation map of the imported module?